curated_transformers/models/bert/config.py

```diff
@@ -1,134 +1,40 @@
 from dataclasses import dataclass
 
-# Defaults taken from syntaxdot
-# https://github.com/tensordot/syntaxdot/blob/22bd3d43ed2d7fcbef8a6217b01684194fae713f/syntaxdot-transformers/src/models/bert/config.rs#L25
+from ..config import (
+    TransformerAttentionConfig,
+    TransformerEmbeddingConfig,
+    TransformerLayerConfig,
+)
 
 
-@dataclass
-class BERTEmbeddingConfig:
+class BERTEmbeddingConfig(TransformerEmbeddingConfig):
```

Comment:
I kept them just in case new, model-specific settings get added in the future, but we can indeed reintroduce the subclasses if and when that happens. I'll remove them.